tests/sentry/api/endpoints/test_organization_dashboards.py

```diff
@@ -455,45 +455,6 @@ def test_post_errors_if_layout_submitted_without_required_keys(self):
         response = self.do_request("post", self.url, data=data)
         assert response.status_code == 400, response.data
 
-    def test_post_widgets_with_null_columns_and_aggregates_succeeds_and_sets_value(self):
```

Comment:
@shruthilayaj I can see why you wanted to do the serializer change in this PR. Removing this test but not requiring columns and aggregates could leave us with a buggy situation.

I changed the serializer to require the `columns` and `aggregates` and 27 tests failed. I can make those changes here